### DISCUSSION TITLE
babyagi plugin

### DISCUSSION THREAD

**Discussion by StableInquest** [oobabooga/text-generation-webui#857] [Ideas]
Is anyone working on a babyagi plugin?(https://github.com/yoheinakajima/babyagi) I haven't researched this at all. Would be great to link babyagi up with LLM of your choice locally an not only OpenAI.

**Comment by eucdee** [comment]
babyagi is less then 200 lines of python code. it shouldnt be that hard to make it work with the api especially with our new helpers.. ai does the coding we humans only need to implement it correctly.. this rollercoaster is about to get real fast...

**Comment by MaiJames** [comment]
Have a look at this:
[BabyAGI closed pull request](https://github.com/yoheinakajima/babyagi/pull/156) merged into this other [pull request](https://github.com/yoheinakajima/babyagi/pull/101)

I also saw this other project:
[BigBoogaAGI](https://github.com/DataBassGit/BigBoogaAGI)

**Comment by mlnima** [comment]
nice to find this topic, I will try it with the wizardLM 7B model which people say is good, so far it can write some frontend react code well, but need a very good and highly detailed prompt. I am about to start now wish me good result :D
